test/functional/ignore-list

```diff
@@ -5,6 +5,8 @@
 ^swupd-client software verify [0-9.]+$
 ^.*Copyright \(C\) .*$
 ^$
+^Downloading packs...
+^Extracting.*
```

Comment:
IMO, any future changes to the pack download code would benefit from *not* ignoring the "Extracting ..." lines, because they print the version now. We always want to make sure we're downloading the correct packs for the correct version (or not downloading them at all).